tools/tpm2_evictcontrol.c

```diff
@@ -64,11 +65,13 @@ struct tpm_evictcontrol_ctx {
         UINT8 c : 1;
         UINT8 P : 1;
     } flags;
+    UINT8 persist_passed : 1;
```

Comment:
Another way could be setting P to two bits and setting it 2 if it was an implied -P, thoughts?

Reply:
I'm not opposed. It seems marginally harder to grok, but not significantly so. Happy to make that change.

Reply:
Consider this optional, you can leave it as you like.